woocommerce/woocommerce-android
# Label purchased screen

Part of: #12288

### Description
This PR adds the final screen after purchasing a shipping label. The changes here only contain the final screen UI and the navigation to that final screen. The links on the screen, buttons and logic in general will be added in a different PR.


### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap on create shipping labels
5. Expand the shipment details bottom sheet
6. Tap on purchase label
7. Check that the app navigates to the final screen
8. Check that UI in the final screen works as expected (navigation and logic will be added in a different PR)

### The tests that have been performed
1. Test that the UI interaction works as expected

### Images/gif

https://github.com/user-attachments/assets/14a0fdea-02dd-407f-b423-f6286496ba96




## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
